core/src/models/Property.ts

```diff
@@ -40,7 +40,7 @@ import { Run } from "./Run";
 import { Source } from "./Source";
 
 const jsMap = {
-  boolean: config.sequelize.dialect === "sqlite" ? "text" : "boolean", // there is no boolean type in SQLite
+  boolean: config?.sequelize?.dialect === "sqlite" ? "text" : "boolean", // there is no boolean type in SQLite
```

Comment:
This is interesting - when running the `./bin/migrator` command, we don't actually have the full actionhero config loaded up.  We did refrence the FilterDictionary in a migration, which then imports Filter, then Property...